tox.ini

```diff
@@ -31,7 +31,7 @@ commands =
 deps =
     reuse
 commands =
-    reuse lint
+    reuse --debug lint
```

Comment:
What does it do? the commit message does not explain why you added it.

Reply:
Don't be too fast :)
I'm still messing around with it, as tox -e reuse is broken.
Seems to be because git ls-files doesn't work when the repository is checked out via action/checkout :)
Now only need to find a way to fix it properly

Reply:
ok, I got mail during a boring meeting so I looked at it :-)

Reply:
If you have another boring meeting :D